### ISSUE TITLE
The attention mask and the pad token id were not set.

### ISSUE THREAD

**Issue by llmwesee** [h2oai/h2ogpt#625]
Sometimes i got the following error:
"The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's attention_mask to obtain reliable results.
Setting pad_token_id to eos_token_id:2 for open-end generation."

For resolving this i want to use the auto_attention argument for automatically generate an attention mask and pad token ID based on the input sequence.
`model = Transformer(auto_attention=True)`
I went through all the files but didn't find where i set this argument so that i want to know where i can set the auto_attention for llama2 model

**Comment by pseudotensor** [comment]
It's not an error and is common, but should have no negative effect.

**Comment by Soumendraprasad** [comment]
@pseudotensor , I am also facing this issue . Is there any way to handle .

**Comment by pseudotensor** [comment]
Hi, it's just a warning.  I've not been able to see how to avoid the warning so far,  or to disable it.
